index.js

```diff
@@ -0,0 +1,23 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+let input = document.querySelector('#input')
+let form = document.querySelector('form')
+let list = document.querySelector('ul')
+let text = document.querySelector('#input')
+
+form.addEventListener("submit", () => {
```

Comment:
If you're going to use `event.preventDefault()` you would need to add `event` as an argument on the callback function